python/aitemplate/backend/cuda/tensor/expand.py

```diff
@@ -159,13 +161,21 @@ def gen_function_decl(func_attrs: Dict[str, Any]) -> str:
     return;
   }
   // Determine stride for each input dimension
+  {% if is_windows %}
+  {{index_type}}* input_strides = ({{index_type}}*) malloc(input_rank * sizeof(int64_t));
```

Comment:
Thanks @hlky !
Here, both input_rank and output_rank are actually constants. They could be codegen-ed ahead-of-time instead of being passed around at runtime. In this case, the malloc cost can be avoided. Do you want to fix in this way?

Reply:
@ipiszy Thanks for the suggestion. input_rank and output_rank are now codegen-ed.